spec/policies/group_policy.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe GroupPolicy do
+  subject { GroupPolicy.new(user, group) }
+
+  let(:group) { FactoryGirl.create(:group) }
+
+  context 'when guest' do
+    let(:user) { nil }
+
+    it { is_expected.to permit_action(:show)    }
+
+    it { is_expected.to forbid_action(:index)   }
+    it { is_expected.to forbid_action(:new)     }
+    it { is_expected.to forbid_action(:create)  }
+    it { is_expected.to forbid_action(:update)  }
+    it { is_expected.to forbid_action(:edit)    }
+    it { is_expected.to forbid_action(:destroy) }
+  end
+
+  context 'when regular group member' do
```

Comment:
And thanks for pointing that out, made me realize I'm missing the `_spec` suffix in the filename...